static/src/javascripts/projects/commercial/modules/dfp/prepare-permutive.js

```diff
@@ -100,8 +117,12 @@ const runPermutive = (
             throw new Error('Global Permutive setup error');
         }
 
-        const payload = generatePayload(pageConfig);
+        const permutiveIdentities = generatePermutiveIdentities(pageConfig);
+        if (permutiveIdentities.length > 0) {
+            permutiveGlobal.identify(permutiveIdentities);
```

Comment:
Should we check whether `permutiveGlobal.identify` is defined as we do for `permutiveGlobal.addon` elsewhere in this function?

Reply:
The previous block is checking that permutive is set up and providing a more helpful error if it is not, if it's there we expect it to be correct. I'm not sure we'd typically check the existence of all our third party module's methods before calling them, although doing so isn't without value in JavaScript.

Be great to chat to you in person to get your advice on the best way to implement a check that provides useful developer feedback.

Reply:
We agreed this isn't required, but we should look at improving the existing check in a separate PR.